lib/jwt/AccessToken.js

```diff
@@ -212,6 +212,7 @@ function AccessToken(accountSid, keySid, secret, options) {
   if (!keySid) { throw new Error('keySid is required'); }
   if (!secret) { throw new Error('secret is required'); }
   options = options || {};
+  if (_.isInteger(options.identity)) { options.identity = new String(options.identity); };
```

Comment:
I'd rather fix the check on line 252 below. Could it just be `grants.identity = String(this.identity);`?